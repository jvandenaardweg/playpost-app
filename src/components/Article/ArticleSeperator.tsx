import React from 'react';
import { View } from 'react-native';

import styles from './styles';

export const ArticleSeperator: React.FC = () => (
  <View style={styles.seperator}></View>
);
