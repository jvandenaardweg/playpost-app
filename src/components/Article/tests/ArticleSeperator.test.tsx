import React from 'react';
import { render, RenderAPI } from 'react-native-testing-library';
import { ArticleSeperator } from '../ArticleSeperator';

describe('ArticleSeperator', () => {

  describe('rendering', () => {
    let wrapper: RenderAPI;

    beforeAll(() => {
      wrapper = render(<ArticleSeperator />);
    });

    it('should render correctly', () => {
      expect(wrapper.toJSON()).toMatchSnapshot();
    });
  });
});
