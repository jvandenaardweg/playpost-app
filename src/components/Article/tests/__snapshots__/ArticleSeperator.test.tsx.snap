// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ArticleSeperator rendering should render correctly 1`] = `
<View
  style={
    Object {
      "backgroundColor": "#ffffff",
      "paddingLeft": 16,
      "paddingRight": 16,
    }
  }
>
  <View
    style={
      Object {
        "borderBottomColor": "#cccccc",
        "borderBottomWidth": 0.5,
      }
    }
  />
</View>
`;
